components/DualMessageOverlay.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Dimensions, View } from 'react-native';
+import { Overlay } from 'react-native-elements';
+import { Column, Row, Text } from './ui';
+import { HalfButton } from './ui/Button';
+import { Theme } from './ui/styleUtils';
+
+/**
+ * DualMessageOverlay is like MessageOverlay but with two buttons
+ *
+ * NOTE: This has been added for surfacing bugs and needs to be refactored
+ * before use.
+ */
+export const DualMessageOverlay: React.FC<DualMessageOverlayProps> = (
+  props
+) => {
+  const { t } = useTranslation('common');
+  return (
+    <Overlay
+      isVisible={props.isVisible}
+      overlayStyle={Theme.MessageOverlayStyles.overlay}
+      onShow={props.onShow}
+      onBackdropPress={props.onBackdropPress}>
+      <Column
+        width={Dimensions.get('screen').width * 0.8}
+        style={
+          !props.progress
+            ? Theme.MessageOverlayStyles.popupOverLay
+            : { height: 100 }
+        }>
+        <Column padding="21" crossAlign="center">
+          {props.title && (
+            <Text
+              align="center"
```

Comment:
move all the styles, to DefaultTheme.ts and PurpleTheme.ts and use Theme object to access the styles here.

Reply:
I'd like to take this as a separate feature/improvement PR as this also exists in MessageOverlay components and I'd put them in a common CSS class attribute.